lib/_tls_wrap.js

```diff
@@ -1098,14 +1098,24 @@ function onConnectEnd() {
   }
 }
 
+let warnOnAllowUnauthorized = true;
+
 // Arguments: [port,] [host,] [options,] [cb]
 exports.connect = function connect(...args) {
   args = normalizeConnectArgs(args);
   var options = args[0];
   var cb = args[1];
+  const allowUnauthorized = '0' === process.env.NODE_TLS_REJECT_UNAUTHORIZED;
```

Comment:
Can we reverse this equality check?: `process.env.NODE_TLS_REJECT_UNAUTHORIZED === '0'`